src/barkhaPJun20/SumOfNumInStringHavingAlphabet.java

```diff
@@ -0,0 +1,17 @@
+package barkhaPJun20;
+
+public class SumOfNumInStringHavingAlphabet {
+	int sum=0;
+	int sumOfNumbers(String numberString) {
+		for (int index=0; index<numberString.length(); index++) {
+			if (Character.isDigit(numberString.charAt(index)))     //'7' =>digit in character form
+				sum+=Integer.parseInt(String.valueOf(numberString.charAt(index)));  //this is to convert char 7 to String 7 because parseInt(convert String 7 to int 7) is applicable to String only.
```

Comment:
You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));